### PR TITLE
Added Category page and changes Home page

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -1,34 +1,25 @@
 body{
-    background: var(--darkgrey);
-    text-align: center;
+    margin: 0px;
```
Comment (AtulHande03a) [inline main.css]: doesn't need px , can write 0

**Review by AtulHande03a — APPROVED** [review]
looks good !!! ✨

File: pages/categories/categories.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <div class="category-page">
+
+        <h1 class="heading "> Categories </h1>
+        <p class="subheading p-md ">Choose any quiz to attempt from the following categories. Ensure that before attempting any quiz, you have gone through
+            the rules from the homepage.</p>
+
+        <!-- <a href="./pages/rules/rules.html ">
+            <button class="btn rules ">Read the rules</button>
+        </a> -->
+
```
Comment (abhishektanwar) [inline pages/categories/categories.html]: remove commented code.

File: main.css [hunk 3]
```diff
@@ -1,34 +1,25 @@
...
-.card-img{
-    max-width: 340px;
-}
-.category-content{
+    color: var(--lightlavender);
     text-align: left;
+    margin-bottom: 2rem;
+}
+.options{
+    width: clamp(300px, 35vw, 900px);
```
Comment (abhishektanwar) [inline main.css]: Question: Any specific reason to use clamp here? In my opinion, this functionality could be achieved with flexbox also.

File: main.css (hunk 1 above)
Comment (shivanigangadharan) [reply main.css]: Done [44fed5b](https://github.com/shivanigangadharan/trivia-bank/pull/4/commits/44fed5bd0337d80ea33664453f5031e68e73a5ed)

**Review by abhishektanwar — APPROVED** [review]
Please check comments . Apart from that code looks clean and good to merge

File: main.css (hunk 3 above)
Comment (shivanigangadharan) [reply main.css]: Was required initially, not anymore with the image added, so have removed it [f5ffa62](https://github.com/shivanigangadharan/trivia-bank/pull/4/commits/f5ffa624cec79c9c3c2a86b001828277f87476e8)

File: pages/categories/categories.html (hunk 2 above)
Comment (shivanigangadharan) [reply pages/categories/categories.html]: Done [a8d32c6](https://github.com/shivanigangadharan/trivia-bank/pull/4/commits/a8d32c6b52998582c8ab40b7863c3ef7828fd68b)

**Pull request merged**